code/dynamic_programming/minimum_cost_polygon_triangulation/MinimumCostPolygonTriangulation.cpp

```diff
@@ -0,0 +1,71 @@
+/* Part of Cosmos by OpenGenus Foundation */
+
+#include <iostream>
+#include <cmath>
+#define MAX 1000000.0
+
+using namespace std;
+ 
+// Structure of a point in 2D plane
+struct Point
+{
+    int x, y;
+};
+ 
+// Utility function to find minimum of two double values
+double min(double x, double y)
+{
+    return (x <= y)? x : y;
```

Comment:
Just nitpicking, the usual condition is `x < y`, although it makes little difference, since if the variables are equal, it doesn't matter which is picked.